### PR TITLE
mainnet updates

### PR DESCRIPTION
**Pull request by stefa2k**
These changes to branch `mainnet` are major and update should be done with caution! Although tested there can be some update issues. Please read this notes carefully before updating your setup!

**Lighthouse:**
- update to version `1.1.0`
- easier launchpad wallet import
- usage of stereum's lighthouse docker images (more secure, easier to use)
- environment files changed, please make sure you adapt these to your needs (only needed if you made changes to them before)
- **usage of new `docker-compose.yaml` is necessary!**

**Lodestar:**
- introducing lodestar for mainnet (although not recommended to use it yet for mainnet)

**Multiclient:**
- usage of stereum's lighthouse docker image (more secure, easier to use)
- environment files for lighthouse beacon changed, please make sure you adapt these to your needs (only needed if you made changes to them before)
- **usage of new `docker-compose.yaml` is necessary!**

**Nimbus:**
- introducing nimbus for mainnet

**Prysm:**
- update to version `1.1.0`
- optimized for new web-ui
- **removed ARM64 support** (this should only be temporarily, we work on building images for ARM64)

**Teku:**
- update to version `20.12.1`

### CODE REVIEW

File: compose-examples/lighthouse-only/README.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
-# Lighthouse standalone fullstack
+# Lighthouse node
 
 ## Requirements
 * Get to know [Lighthouse](https://lighthouse-book.sigmaprime.io/) a bit
 * Server with 4 (v)cpus & 8 gb memory & 100 gb storage
 
 ## Services
-* geth (beacon conntects to it to see deposits for validators)
+* [geth](https://github.com/ethereum/go-ethereum) (beacon conntects to it to see deposits for validators)
```
Comment (gbayasgalan) [inline compose-examples/lighthouse-only/README.md]: conntects!!

**Review by gbayasgalan — APPROVED** [review]

File: compose-examples/lighthouse-only/README.md (hunk 1 above)
Comment (stefa2k) [reply compose-examples/lighthouse-only/README.md]: I think that's an outdated commit, this should be resolved and I can't find it anymore 🤔

**Review by gbayasgalan — APPROVED** [review]

**Review by gbayasgalan — APPROVED** [review]

**Review by gbayasgalan — APPROVED** [review]

**Review by gbayasgalan — APPROVED** [review]

File: config/lighthouse/start-beacon-node.sh [hunk 2]
```diff
@@ -25,7 +25,7 @@ exec lighthouse \
         --debug-level $DEBUG_LEVEL \
         --network $NETWORK \
         beacon_node \
-        --eth1-endpoint $VOTING_ETH1_NODE \
+        --eth1-endpoints $VOTING_ETH1_NODE \
```
Comment (gbayasgalan) [inline config/lighthouse/start-beacon-node.sh]: <start-beacon-node.sh> doesn't exist anymore, right?

**Review by gbayasgalan — APPROVED** [review]

**Review by gbayasgalan — APPROVED** [review]

File: compose-examples/lighthouse-only/README.md (hunk 1 above)
Comment (gbayasgalan) [reply compose-examples/lighthouse-only/README.md]: okey, got it

File: config/lighthouse/start-beacon-node.sh (hunk 2 above)
Comment (stefa2k) [reply config/lighthouse/start-beacon-node.sh]: correct, the startup script is now inside the container, look here for details: https://github.com/stereum-dev/ethereum2-docker-build/tree/main/lighthouse/start

I think this is again an outdated commit you see

**Review by gbayasgalan — APPROVED** [review]

File: config/lighthouse/start-beacon-node.sh (hunk 2 above)
Comment (gbayasgalan) [reply config/lighthouse/start-beacon-node.sh]: ah yes, thank you

**Review by gbayasgalan — APPROVED** [review]

**Pull request merged**